travel-event-host-app/src/lib/context.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+import { useSession } from 'next-auth/react';
+
+import React, { useEffect, useState } from 'react';
+
+const OnboardingContext = React.createContext<any>(undefined);
```

Comment:
Just for consistency in capitalization
```suggestion
const onboardingContext = React.createContext<any>(undefined);
```